contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,133 @@
+
+[John Egan](https://jwegan.com/growth-hacking/managing-growth-teams-portfolio-step-step-guide/), former head of growth engineering at Pinterest, recommends two levers you can pull:
+
+1. Increase the number of high-quality project ideas.
+2. Maximize the impact of every experiment you ship.
+
+### Increase the number of high-quality project ideas
+
+The best way to do this is to create a bottom-up culture where it's the job of **every single person** to come up with ideas. That means every engineer, designer, data scientist, sales rep, marketer, etc.
+
+When it's everyone's job to do this, the number of ideas increases dramatically. Then, by applying the EVENLYN framework mentioned above, you can distill the list to only the best ideas.
+
+> **💡 Tip:** To create an inclusive ideas culture, make your [brainstorms asynchronous](https://source.opennews.org/articles/redesigning-brainstorming-asynchronous/).
+
+### Maximize the impact of every experiment you ship
+
+Every engineer must own the experiments they are working on. They are the ones responsible for making them as impactful as possible – even if it wasn't originally their idea! 
+
+In practice, this means they should work to improve the initial hypothesis and develop further iterations to test. This also means allowing them to fail without being overly harsh on them.
+
+## Problem 4: Your team is afraid of making mistakes
+
+If your team is not making mistakes, you're not moving fast enough. 
+
+In fact, it's not uncommon for growth teams to cause major incidents! And their goal must not be to avoid them.
+
+"There's an anecdote that when airbags were introduced in cars, the average speed on the highway went up. This is essentially what you want to do for your growth team. Give them the freedom to make mistakes, but make sure they resolve them quickly."
+
+### How to fix it
+
+Your aim should be to build a resilient system that can rapidly recover from mistakes or incidents.
+
+In the context of growth teams, there are a few ways to do this:
+
+- Monitor critical metrics and set up alerts. Ensure you can quickly roll back changes using [configurators or feature flags](https://posthog.com/blog/feature-flags-vs-configuration).
+- Set up [review apps](https://seanconnolly.dev/review-applications) so your team can easily test and dogfood your experiments.
+- To test releases in production, set up [conditional overrides](/docs/experiments/testing-and-launching) so you can quickly test each experiment variant.
+- Instead of crashing your app when rendering errors occur, display a fallback UI. In React, you can easily add these with [Error Boundaries](https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary) (and similar concepts existing in other frameworks).
+
+Notice that none of the solutions involve writing tests, which are a **waste of time** for experiments that may only last for a week.
```

Comment:
First, I think this is the key quote: https://twitter.com/rauchg/status/807626710350839808

Automated tests "are a waste of time" is probably too aggressive.   They're expensive overhead - valuable, but not good ROI so long as you are (a) executing tests manually and (b) have enough other safeguards in place.

Even still, every company has a unique set of criteria and needs to figure out whether automated tests for individual experiments make sense for them.  All I would add here is "consider what it would take to not write tests for individual short-lived experiments"

Reply:
changed to "can be a waste of time" and added "Instead, consider what it would take to comfortably ship changes without tests."